tests/python/contrib/test_hexagon/metaschedule_e2e/test_resnet50_int8.py

```diff
@@ -184,3 +187,177 @@ def test_resnet50(hexagon_launcher):
+                        sch.compute_inline(post_block)
+                    next_post_blocks += next_consumers
+                if len(next_post_blocks) == 0:
+                    assert len(post_blocks) == 1
+                    outer_block = post_blocks[0]
+                    break
+                post_blocks = next_post_blocks
+        else:
+            outer_block = conv2d_block
+
+        # Move the conv2d mma into the injective post mma compute block
+        if outer_block != conv2d_block:
+            loops = sch.get_loops(outer_block)
+            # TODO(csullivan): Currently does all post conv2d mma steps
+            # directly after accumulation for one spatial pixel. May
+            # be desirable to do this with coarser spatial granularity
+            sch.compute_at(conv2d_block, loops[4])
+
+        def index_map_nchw32c_nchw8h8w32c(n, c, h, w, c32):
+            return [n, c, h // 8, w // 8, h % 8, w % 8, c32]
+
+        # Add cache for input and output activation layout transform,
+        # note that weight is already in correct layout
+        input_cache = sch.cache_read(conv2d_block, 0, "global")
+        output_cache = sch.cache_write(outer_block, 0, "global")
+        # Transform the layout of the input
+        sch.transform_layout(
+            conv2d_block, ("read", 0), index_map=index_map_nchw32c_nchw8h8w32c, pad_value=0
+        )
+        # Transform the layout of the int32 accumulator
+        sch.transform_layout(
+            conv2d_block, ("write", 0), index_map=index_map_nchw32c_nchw8h8w32c, pad_value=0
+        )
+        # Transform the layout of the output
+        sch.transform_layout(
+            outer_block, ("write", 0), index_map=index_map_nchw32c_nchw8h8w32c, pad_value=0
+        )
+        return True
+
+    return schedule_fn
+
+
+def tune_packed_8x8x32_template(mod, params, hexagon_launcher):
+    def schedule_rule_conv2d_packed_8x8x32(sch: Schedule, conv2d_block: BlockRV):
+        _schedule_packed_8x8x32_conv2d(do_tune=True)(sch, conv2d_block)
+        return [sch]
+
+    # register_func("meta_schedule.conv2d_NCHWc_int8", schedule_rule_conv2d_packed_8x8x32)
+
+    def schedule_conv2d_for_tune(sch: Schedule):
+        _schedule_packed_8x8x32_conv2d(do_tune=True)(sch)
+
+    # This line is necessary for link-params to take effect during
+    # task extraction and relay.build(...).
+    mod = mod.with_attr("executor", executor)
+
+    with tempfile.TemporaryDirectory() as work_dir:
+        database = ms.relay_integration.tune_relay(
+            mod=mod,
+            target=target,
+            params=params,
+            work_dir=work_dir,
+            max_trials_global=20000,
+            max_trials_per_task=1,
+            num_trials_per_iter=1,
+            strategy="replay-trace",
+            builder=get_hexagon_local_builder(),
+            runner=get_hexagon_rpc_runner(hexagon_launcher, number=20),
+            # TODO(csullivan): Configrm the below is accurate
```

Comment:
cc @zxybazh so by using `ScheduleFn` here, MS won't be looking at non conv2d blocks?

Reply:
By using `ScheduleFn` here, the search space is solely generated by the schedule function over the whole workload. In this case, if you're using `_schedule_packed_8x8x32_conv2d`'s `schedule_fn` to construct the `ScheduleFn` space generator, it will only look at conv2d blocks because the function doesn't transform any of the non-conv2d blocks right?

Reply:
Yeah that's my understanding.

Reply:
@csullivan so shall we remove the TODO comment (which also has a typo lol)

Reply:
:) Yes, but one more question, if one utilizes a `ScheduleFn` to define a search space, but also sets custom `sch_rules` to be applied per block as, 
```
            space=ms.space_generator.ScheduleFn(
                schedule_conv2d_for_tune,
                sch_rules=[some set of rules] , # <<<<
                postprocs=[],
                mutator_probs={},
            ),
```
what will be the interaction between the scheduling function and the rules?

Reply:
For `ScheduleFn` I don't think there will be any interaction between the scheduling function and the rules, they will only be used for `PostOrderApply` space generator.

Reply:
@csullivan can you remove the TODO comment?